spec/factories/keys.rb

```diff
@@ -0,0 +1,5 @@
+FactoryGirl.define do
+  factory :key do
+    number 1
```

Comment:
Make this a sequence. See: https://github.com/thoughtbot/factory_girl/blob/master/GETTING_STARTED.md#sequences

```ruby
sequence :number { |n| n }
```